tests/src/Unit/Service/IdentityManagement/UserSessionTest.php

```diff
@@ -117,38 +117,14 @@ public function testAcquireReset()
         /** @var LoggerInterface|\PHPUnit_Framework_MockObject_MockObject $logger */
         $logger = $this->getMockBuilder(LoggerInterface::class)
             ->getMock();
-        $logger->expects($this->any())->method('info')
+        $logger->expects($this->any())->method('debug')
             ->withConsecutive(['Successfully acquired the "{resource}" lock.'],
                 ['Expiration defined for "{resource}" lock for "{ttl}" seconds.'],
-                ['Retrieved a new mpx token {token} for user {username} that expires on {date}.', $this->callback(function ($context) {
-                    try {
-                        $this->assertArraySubset([
-                            'token' => 'TOKEN-VALUE',
-                            'username' => 'mpx/USER-NAME',
-                        ], $context);
-                        $this->assertMatchesRegularExpression('!\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\+\d{4}!', $context['date']);
-                    } catch (ExpectationFailedException) {
-                        return false;
-                    }
-
-                    return true;
-                })],
                 ['Successfully acquired the "{resource}" lock.'],
-                ['Expiration defined for "{resource}" lock for "{ttl}" seconds.'],
-                ['Retrieved a new mpx token {token} for user {username} that expires on {date}.', $this->callback(function ($context) {
-                    try {
-                        $this->assertArraySubset([
-                            'token' => 'TOKEN-VALUE',
-                            'username' => 'mpx/USER-NAME',
-                        ], $context);
-                        $this->assertMatchesRegularExpression('!\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\+\d{4}!', $context['date']);
-                    } catch (ExpectationFailedException) {
-                        return false;
-                    }
-
-                    return true;
-                })]
-            );
+                ['Expiration defined for "{resource}" lock for "{ttl}" seconds.']);
+        $logger->expects($this->any())->method('info')
```

Comment:
Similar to the comment above, if we log this event with `debug` level, test changes simplify a lot.